cli/src/crypto.spec.js

```diff
@@ -1,17 +1,33 @@
 const expect = require('expect');
-const { encrypt, decrypt } = require('./crypto');
+const crypto = require('crypto');
+const { encrypt, decrypt, KEY_BYTE_LENGTH } = require('./crypto');
 
 describe('Crypto Features', () => {
     it('should keep the consistancy between encryption & decryption', () => {
         const data = 'SOME VERY PRIVATE INFO';
-        const passphrase = 'passphrase';
-        const algo = 'aes-256-ctr';
-        const encryptedData = encrypt(data, passphrase, algo);
+        const privateKey = crypto.randomBytes(KEY_BYTE_LENGTH);
```

Comment:
Use `generateNewPrivateKey` and encrypt from string instead of buffer (just the same as the real case), seems to make `bufferize` method useless.